Grillbot/Modules/CReferenceModule.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading.Tasks;
+using Discord.Commands;
+using Grillbot.Exceptions;
+using Grillbot.Services;
+using Grillbot.Services.Preconditions;
+
+
+namespace Grillbot.Modules
+{
+    [IgnorePM]
+    [Name("Hledani dokumentace")]
+    [RequirePermissions("CReference")]
+    public class CReferenceModule : BotModuleBase
+    {
+        private CReferenceService Service { get; }
+
+
+        public CReferenceModule(CReferenceService service)
+        {
+            Service = service;
+        }
+
+        [Command("cref")]
+        [Summary("Najde cpprefence stranku hledaneho tematu")]
+        public async Task FindCReferenceAsync(string topic)
+        {
+            try
+            {
+                await ReplyAsync(await CReferenceService.GetReferenceUrl(topic));
+            }
+            catch (NotFoundException e)
+            {
+                await ReplyAsync(e.Message);
+            }
+            catch (Exception e)
```

Comment:
This is unnecessary. Other exceptions are automatically sent to log event in Services.BotLoggingService.OnLogAsync.

Reply:
Fixed